Cargo.toml

```diff
@@ -21,4 +21,3 @@ codegen-units = 1
 [patch.crates-io]
 zcash_encoding = { path = "components/zcash_encoding" }
 zcash_note_encryption = { path = "components/zcash_note_encryption" }
```

Comment:
Can we get rid of the path-based dependencies here? IIRC right now we're depending upon the released versions of these crates in `zcash_primitives` and the other crates.